### PR TITLE
new kenny configuration

### PR DESCRIPTION
**Pull request by ipa-nhg**

### CODE REVIEW

File: robotino_bringup/env.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+export MY_CATKIN_WORKSPACE=~/catkin_ws
+source /etc/cob.bash.bashrc
```
Comment (bajo) [inline robotino_bringup/env.sh]: I do not know about this file? Is this something careobot specific?

File: robotino_bringup/robots/tuw-robotino2/launch/tuw-robotino2.launch [hunk 2]
```diff
@@ -15,96 +17,114 @@
...
 
-  <node name="sensing" pkg="squirrel_sensing_node" type="sensing" args="/dev/ttyFingertips em1" output="screen"/>
+	</group>
 
-  <!--node name="wrist_safety" pkg="wrist_safety" type="wrist_safety" output="screen" /-->
+	<group>
+	  <machine name="scrat" address="$(arg hostname_scrat)" env-loader="$(arg env-script)" default="true" timeout="30"/>
 
-  <!--node name="squirrel_view_controller" pkg="robotino_driver" type="squirrel_view_controller" output="screen"/-->
+    <!-- kinect -->
```
Comment (bajo) [inline robotino_bringup/robots/tuw-robotino2/launch/tuw-robotino2.launch]: Should we have the kinect section in both the scrat and robotino group?

Comment (NadiaHG) [reply robotino_bringup/robots/tuw-robotino2/launch/tuw-robotino2.launch]: theoretically only for scrat but right the pc is still not mounted and kinect still connected to robotino, doesn't it?

File: robotino_bringup/env.sh (hunk 1 above)
Comment (NadiaHG) [reply robotino_bringup/env.sh]: right, I forgot this, this is just to change the color of the bash setup and difference the pcs , not needed...

File: robotino_bringup/robots/tuw-robotino2/launch/tuw-robotino2.launch (hunk 2 above)
Comment (bajo) [reply robotino_bringup/robots/tuw-robotino2/launch/tuw-robotino2.launch]: Ok, now I understand. 
Yes, the pc is not mounted yet and the kinect is still on robotino.

**Comment by bajo** [comment]
LGTM. Thanks.

**Pull request merged**